w3af/core/controllers/daemons/tests/test_webserver.py

```diff
@@ -19,6 +19,7 @@
 Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 
 """
+import pytest
```

Comment:
In some cases you're adding `import pytest` and you really don't need it.

Maybe modify the script you wrote to avoid this?

Reply:
script was modified few times between runs and right now it's not a one-time run script, but I was using it incrementally due to some problems. I would like to not run it again ;). But I can check for unused pytest imports